recipes/kalpy/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 70884c243c2eb44423e748af18c9c9a0730e614e88bc4fe7dccade945fe69111
+
+build:
+  number: 0
+  skip: True  # [py<38]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python                                  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}      # [build_platform != target_platform]
+    - cmake
+    - make                                    # [not win]
+    - ninja
+    - pybind11
+    - setuptools
+    - setuptools_scm
+  host:
+    - python
+    - pip
+    - kaldi
+    - openfst
+  run:
+    - python
+    - pynini
+    - kaldi
+    - numpy >=1.x
```

Numpy is _only_ a run-dependence? Also, you cannot use "1.x" (or, at least, I have no idea how conda would interpret that, it's not a valid version). Conda-forge is already globally building only for numpy>=1.21 anymore, anything below that would not be installed without forcing it anyway.
```suggestion
    - numpy
```